frontend/packages/metal3-plugin/src/components/host-menu-actions.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
```

Comment:
Nit: external libs imports (listed first) should be separated from app/pkg imports (listed second) via newline.